### PR TITLE
Add Compliance API landing page

### LINKED ISSUE #1689: Add Compliance API Product Page

### PR DESCRIPTION
**Pull request by michellusthof**
- [x] Create landing page file
- [x] Add content
- [x] Create call-to-action
- [x] Add stylesheet
- [x] Add header image
- [x] Add PDF to downloads
- [x] Replace right image 
- [x] Replace PDF with final version

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
This is a product page?

**Comment by michellusthof** [comment]
> This is a product page?

It's a landing page until we create a proper product page for it. Let's redirect this page to the product page once we have it ready.

**Comment by RonaldDijkstra** [comment]
Smart move

File: source/localizable/compliance-api.html.erb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+<section>
+  <div class="row">
+      <div class="col">
+        <article>
+          <%= partial "compliance-api/content" %>
+        </article>
+      </div>
+      <div class="col-img">
+      <%= image_tag "screenshots/compliance-api-toepassingen.png", title: "Toepassingen van de Compliance API", alt: "Toepassingen van de Compliance API" %>
```
Comment (RonaldDijkstra) [inline source/localizable/compliance-api.html.erb]: indentation

**Comment by RonaldDijkstra** [comment]
Screenshot 2019-03-27 at 11 40 53

Does not fit

File: source/localizable/call-to-actions/_information-compliance-api.erb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    <h2>
+      Geïnteresseerd in meer informatie over de Compliance API?
+    </h2>
+
+    <p>
+      Download ons handige product sheet met alle informatie.
+    </p>
+
+    <form class="form-inline" accept-charset="UTF-8" action="<%= t("urls.contact") %>" method="POST">
```
Comment (RonaldDijkstra) [inline source/localizable/call-to-actions/_information-compliance-api.erb]: For now this suits, but I would rather like to use `partials/form`. But right now it doesn't do 'downloads'.

**Pull request merged**